.commitlintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+  "extends": [
+    "@commitlint/config-conventional"
+  ],
+  "rules": {
+    "type-enum": [
+      2,
+      "always",
+      [
+        "docs",
+        "resources",
+        "planner",
+        "config",
+        "ci",
+        "api",
+        "workflows",
+        "controllers",
+        "tests",
+	"test utils",
```

Comment:
Oh, do we really have to keep an explicit list of directory names in this configuration? Is there no option to allow any valid directory name *plus* a list of additional topics?

Reply:
> Oh, do we really have to keep an explicit list of directory names in this configuration? Is there no option to allow any valid directory name _plus_ a list of additional topics?
good suggestion! I don't know yet if commitlint supports this already but I can research a bit more ...

Reply:
I researched a bit but did not find a way to achieve this. Here is the documentation of rules:

https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md